tests/statistic_reporter_test.py

```diff
@@ -0,0 +1,43 @@
+import time
+import unittest
+from unittest.mock import MagicMock
+
+from forwarder.application_logger import setup_logger
+from forwarder.statistics_reporter import StatisticsReporter
+
+
+class TestStatisticReporter(unittest.TestCase):
```

Comment:
Should be pytest style not unittest

Reply:
Done. just for curiosity. Why not unittest style?

Reply:
Standardisation mostly, we use Pytest for the other projects.
And it is better ;)

Reply:
alright, point taken. :)